server/memcached/src/main/java/org/infinispan/server/memcached/MemcachedTextDetector.java

```diff
@@ -51,7 +53,9 @@ private void installHandler(ChannelHandlerContext ctx) {
       // Add the Memcached server handler
       ctx.pipeline().addLast(server.getInitializer(MemcachedProtocol.TEXT));
       Log.SERVER.tracef("Detected Memcached text connection %s", ctx);
+      ctx.fireChannelRegistered();
```

Comment:
I don't think we want to invoke this. Just let netty do that. The point of the custom event is for handlers we install after channel registration so we don't have to invoke the registration a second time across all the pipeline handlers.

Reply:
I would remove across all of them. We shouldn't need to invoke the channel methods directly, that is what the custom event is for.

Reply:
Updated this one. It _should_ still be fine. Let's see if something shows in the server/tests.